modules/machine-health-checks-baremetal.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * machine_management/deploying-machine-health-checks.adoc
+// * post_installation_configuration/node-tasks.adoc
+
+[id="machine-health-checks-baremetal_{context}"]
+= MachineHealthChecks on Bare Metal
+
+Machine deletion on bare metal cluster triggers reprovisioning of a bare metal host.
```

Comment:
I think the bulk of the content needs to go in modules/machine-health-checks-about.adoc around line 30.